### DISCUSSION TITLE
Still can not access the student benefits

### DISCUSSION THREAD

**Discussion by simhanson123** [community/community#142365] [GitHub Education]
### Hi everyone,

My request had been approved.But I still not able to access the benefits...

**Comment by kleqing** [comment]
As my remembered, you have to wait for 1-2 days or 1 week to get the student benefits!

**Comment by davevad93** [comment]
Hi @simhanson123 ,

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @simhanson123 was approved and your benefits are now available.`

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
